### ISSUE TITLE
We should have Maven format our code

### ISSUE THREAD

**Issue by Christopher-Chianelli** [oldterns/VileBot#88]
Since 99% of problems from code reviews deal with formatting, and all IDE's have different formatters, we should make Maven do it for us, like so:
```
      <plugin>
        <groupId>com.coveo</groupId>
        <artifactId>fmt-maven-plugin</artifactId>
        <version>2.1.0</version>
        <executions>
          <execution>
            <goals>
              <goal>check</goal>
            </goals>
          </execution>
        </executions>
</plugin>
```
running `mvn fm-maven-plugin:format` will format the code. See https://github.com/coveo/fmt-maven-plugin for details. (Note only google formatter (https://google.github.io/styleguide/javaguide.html) is supported). Maven Checkstyle can also work, but would require contributors to set up their IDE's.

**Comment by Christopher-Chianelli** [comment]
https://github.com/revelc/formatter-maven-plugin is another good option. This one can use any eclipse formatter.

**Comment by amisevsk** [comment]
@Christopher-Chianelli Can this issue be closed, now that #91 is merged?

**Comment by Christopher-Chianelli** [comment]
@amisevsk thanks for reminding me. Closing.
